workflows/sars-cov-2-variant-calling/sars-cov-2-pe-illumina-artic-ivar-analysis/pe-wgs-ivar-analysis-test.yml

```diff
@@ -0,0 +1,35 @@
+- doc: Test workflow execution using paired end illumina ARTIC accession
+  job:
+    Reference FASTA:
+      class: File
+      location: 'https://zenodo.org/record/4944064/files/MN908947_3_Wuhan-Hu-1.fasta'
+      filetype: fasta
+    ARTIC Primer BED:
+      class: File
+      location: 'https://zenodo.org/record/4944064/files/SARS-CoV-2-ARTICv3.bed'
+      filetype: bed
+    Amplicon Primer Info:
+      class: File
+      location: 'https://covid19.galaxyproject.org/artic/ARTIC_SARS_CoV-2_amplicon_info_v3.tsv'
```

Comment:
This file is needed for the -f option of ivar trim. You should probably make use of this option also in the training material, add the file to zenodo record 4944064, and get all test data from the same source then.

Reply:
@pvanheus I think that comment still applies ?

Reply:
The Amplicon Primer Info file is now computed (by the `ivar trim` tool) so doesn't need to be provided by the user.